opticalFlow/lucasKanadeOpticalFlow.py

```diff
@@ -25,106 +24,160 @@
 cap = cv.VideoCapture(vidpath)
 
 # get the first frame
 _, old_frame = cap.read()
-old_gray = cv.cvtColor(old_frame, cv.COLOR_BGR2GRAY)
+old_gray = cv.cvtColor(
+    old_frame, cv.COLOR_BGR2GRAY
+)
 
 # get resolution of video
 res_x = len(old_frame[0])
 res_y = len(old_frame)
 
 # create crosshair mask
-crosshair_bottom = int(0.7*res_y)
-crosshair_top = int(0.3*res_y)
-crosshair_left = int(0.3*res_x)
-crosshair_right = int(0.7*res_x)
-crosshairmask = np.zeros(old_frame.shape[:2], dtype="uint8")
-cv.rectangle(crosshairmask, (crosshair_left, crosshair_top), (crosshair_right, crosshair_bottom), 255, -1)
+crosshair_bottom = int(0.7 * res_y)
+crosshair_top = int(0.3 * res_y)
+crosshair_left = int(0.3 * res_x)
+crosshair_right = int(0.7 * res_x)
+crosshairmask = np.zeros(
+    old_frame.shape[:2], dtype="uint8"
+)
+cv.rectangle(
+    crosshairmask,
+    (crosshair_left, crosshair_top),
+    (crosshair_right, crosshair_bottom),
+    255,
+    -1,
+)
 
 # create masks for drawing purposes
-trail_history = [[[(0,0), (0,0)] for i in range(trailLength)] for i in range(numPts)]
+trail_history = [
+    [
+        [(0, 0), (0, 0)]
+        for i in range(trailLength)
+    ]
+    for i in range(numPts)
+]
 
 # get features from first frame
-print(f"\nRunning Optical Flow on: {vidpath}")
-old_points = cv.goodFeaturesToTrack(old_gray, maxCorners=numPts, mask=crosshairmask, **shitomasi_params)
+print(
+    f"\nRunning Optical Flow on: {vidpath}"
+)
+old_points = cv.goodFeaturesToTrack(
+    old_gray,
+    maxCorners=numPts,
+    mask=crosshairmask,
+    **shitomasi_params,
+)
 
 # if saving video
 if savevid:
     # path to save output video
-    pathparts = vidpath.split('.')
-    savepath = '.'+ vidpath.split('.')[-2] + '_LK_FLOW' + '.mp4'
-    print(f"Saving Output video to: {savepath}")
+    pathparts = vidpath.split(".")
+    savepath = (
+        "."
+        + vidpath.split(".")[-2]
+        + "_LK_FLOW"
+        + ".mp4"
+    )
+    print(
+        f"Saving Output video to: {savepath}"
+    )
 
     # get shape of video frames
     height, width, channels = old_frame.shape
 
     # setup videowriter object
-    fourcc = cv.VideoWriter_fourcc(*'mp4v')
-    videoOut = cv.VideoWriter(savepath, fourcc, fps, (width, height))
-
+    fourcc = cv.VideoWriter_fourcc(*"mp4v")
+    videoOut = cv.VideoWriter(
+        savepath,
+        fourcc,
+        fps,
+        (width, height),
+    )
 # PROCESS VIDEO ---------------------------------------------------------------
-while(True):
+while True:
     # get next frame and convert to grayscale
     stillGoing, new_frame = cap.read()
 
     # if video is over, quit
     if not stillGoing:
         break
-
     # convert to grayscale
-    new_frame_gray = cv.cvtColor(new_frame, cv.COLOR_BGR2GRAY)
+    new_frame_gray = cv.cvtColor(
+        new_frame, cv.COLOR_BGR2GRAY
+    )
 
     # calculate optical flow
-    new_points, st, err = cv.calcOpticalFlowPyrLK(old_gray, new_frame_gray, old_points, None, **LK_params)

```

Comment:
I feel like spreading this out over multiple lines makes it really unreadable for tuple unpacking